app/components/dropdown_menu_component.rb

```diff
@@ -26,6 +27,6 @@ def render?
   end
 
   def button_label
-    content_tag(:span, @menu_title, class: @hide_label ? "sr-only" : nil)
+    content_tag(:span, @menu_title, class: @hide_label ? "sr-only" : "mr-5")
```

Comment:
```suggestion
    content_tag(:span, @menu_title, class: ["mr-5", {"sr-only" => @hide_label}])
```

could do this via https://edgeapi.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-token_list